packages/app/obojobo-document-engine/src/scripts/viewer/components/viewer-app.js

```diff
@@ -236,82 +241,82 @@ export default class ViewerApp extends React.Component {
 			if (loadingEl && loadingEl.parentElement) {
 				document.getElementById('viewer-app').classList.add('is-loaded')
 				loadingEl.parentElement.removeChild(loadingEl)
-				this.scrollToTop()
+				// this.scrollToTop()
```

Comment:
The scroll to top needs to be tweaked. Changing pages on master instantly changes page (with no scrolling) to the end user. The redux implementation currently animate scrolls up to the top.